Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -201,4 +202,9 @@ public double convertTicksToInches(double ticks) {
     public double convertInchesToTicks(double inches) {
         return inches * ticksPerInch.get();
     }
+
+    @Override
+    public void updateTelemetry() {
+        this.updateMotorTelemetry();
```

Comment:
I'd made a comment about this at the top of the PR:
Wrapped the existing updateMotorTelemetry() since there are some other outstanding changes to it in PR and wanted to minimize merge conflicts.

I think the dependent changes have gone in though since I created this PR so can do the refactoring now.